gef-ui/templates/partials/amendments/facility-value.njk

```diff
@@ -0,0 +1,83 @@
+{% extends "index.njk" %}
+{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/input/macro.njk" import govukInput %}
+
+{% block pageTitle -%}
+  New facility value
+{%- endblock %}
+
+{% set headingHtml %}
+  <header>
+    <span class="govuk-caption-l" data-cy="heading-caption">
+      {{ exporterName }}
+    </span>
+    <h1 class="govuk-label-wrapper">
+      <label class="govuk-label govuk-label--xl govuk-!-padding-bottom-3" for="facility-value">
```

Comment:
Could this not be part of the govukInput component by using isPageHeading?

`label: {
              text: 'New facility value',
              isPageHeading: true,
              classes: "govuk-label--xl",
              attributes: {
                'data-cy': 'label-heading'
              }
            },`

Like in gef-ui\templates\partials\facilities.njk which also has a heading caption

Reply:
AFAIK the `exporterName` needs to be part of the same `header` group as the `h1` to be semantically accurate - I'm not aware of any way of including the caption otherwise